diesel/src/migration/mod.rs

```diff
@@ -0,0 +1,55 @@
+//! Representation of migrations
+
+pub mod errors;
+pub use self::errors::*;
+
+use connection::SimpleConnection;
+use std::path::Path;
+
+/// Represents a migration that interacts with diesel
```

Comment:
Just noticed that's it's super hard to see what's up with SimpleConnection – it's not public and the only way aside from reading the code where you can find it is in the docs for Connection (which it's a supertrait of). Not something for this PR to fix but seemed worth commenting on.